app/model/editions/templates/feast/FeastHelpers.scala

```diff
@@ -0,0 +1,11 @@
+package model.editions.templates.feast
+
+import model.editions.CuratedPlatform
+import model.editions.templates.CuratedPlatformWithTemplate
+
+trait FeastEdition extends CuratedPlatformWithTemplate {
```

Comment:
Should this be `trait FeastAppDefinition`?

Reply:
👍  caf6115a1c